ws-nextjs-app/pages/[service]/live/[id]/Header/LiveLabelHeader/index.styles.tsx

```diff
@@ -0,0 +1,30 @@
+import { css, Theme } from '@emotion/react';
+
+const styles = {
+  liveLabelPulse: ({ spacings, palette }: Theme) =>
+    css({
+      width: `${spacings.TRIPLE}rem`,
+      height: `${spacings.TRIPLE}rem`,
+      color: palette.LIVE_LIGHT,
+    }),
+  liveLabelTextWithImage: ({ palette }: Theme) =>
+    css({
+      'span:first-child': { color: palette.LIVE_LIGHT },
+    }),
+  liveLabelTextWithoutImage: ({ mq, palette }: Theme) =>
+    css({
+      'span:first-child': {
+        display: 'inline-flex',
+        color: palette.LIVE_LIGHT,
+        'overflow-wrap': 'anywhere',
+        [mq.GROUP_4_MIN_WIDTH]: {
+          width: 'calc(100% / 3  - 2.25rem)',
```

Comment:
Noticed a few of these calculations in a couple of header related components. Is there any way to use more constant values? Feels like little magic number-y, but perhaps needed to get it to match the design spec?

Reply:
Yeah I did this because of recurring percentages like 33.3333..., is there a succinct way to define that value, can i just use 33.33%?

Reply:
Alternative is to break down the LiveLabel compound component even more, so we can use Flex properly

Reply:
I think you could just do `calc(100% / 3)` and let the browser handle the precision.

Reply:
Ahh sorry tired office brain, the 2.25 rem being the magic number, yeah I pulled to cater to padding somewhere, lemme see if I can make it a constant to make it easier